src/wp-includes/class-phpass.php

```diff
@@ -34,243 +35,326 @@
  * @since 2.5.0
  */
 class PasswordHash {
-	var $itoa64;
-	var $iteration_count_log2;
-	var $portable_hashes;
-	var $random_state;
 
 	/**
-	 * PHP5 constructor.
+	 * Itoa64.
+	 *
+	 * @var string
 	 */
-	function __construct( $iteration_count_log2, $portable_hashes )
-	{
+	private $itoa64;
+
+	/**
+	 * Iteration Count Log2.
+	 *
+	 * @var integer
+	 */
+	private $iteration_count_log2;
+
+	/**
+	 * Portable Hashes.
+	 *
+	 * @var boolean
+	 */
+	private $portable_hashes;
+
+	/**
+	 * Random State.
+	 *
+	 * @var string
+	 */
+	private $random_state;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param integer $iteration_count_log2 Iteration Count Log2.
+	 * @param boolean $portable_hashes      Portable Hashes.
+	 */
+	public function __construct( int $iteration_count_log2, bool $portable_hashes ) {
```

Comment:
WordPress has a minimum version requirement of PHP 5.6, so `bool` and `int` type hints in functions can't be used, since they're PHP 7+ only.

Reply:
Ok, all type hints should be removed.